src/Benchmarks/Startup.cs

```diff
@@ -319,6 +325,18 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 app.UseMvc();
             }
 #elif NETCOREAPP3_0
+            if (Scenarios.Any("EndpointPlaintext"))
+            {
+                app.UseRouting();
+
+                app.UseEndpoints(endpoints =>
+                {
+                    endpoints.MapGet(
```

Comment:
Does this look correct?

Reply:
How close should this be to the other plain text benchmark?

The other benchmark writes text bytes:
https://github.com/aspnet/Benchmarks/blob/aa253347ad9a41365446d5706e81c7eb166e61d9/src/Benchmarks/Middleware/PlaintextMiddleware.cs#L37-L41

Routing wise, this is doing a MapGet which the other benchmark doesn't care about the HTTP verb. Also "/ep/plaintext" has slightly more routing overhead compared to "/ep-plaintext"

Reply:
Updated to make it more like PlainTextMiddleware: 

| Description |       RPS | CPU (%) | Memory (MB) | Avg. Latency (ms) | Startup (ms) | First Request (ms) | Latency (ms) |
| ----------- | --------- | ------- | ----------- | ----------------- | ------------ | ------------------ | ------------ |
|             | 2,192,976 |      98 |          67 |              1.23 |          273 |              95.38 |         0.67 |